easybuild/easyconfigs/j/jax/jax-0.2.18-correct_libraries.patch

```diff
@@ -0,0 +1,63 @@
+diff -ruN jax_orig/build/build.py jax/build/build.py
+--- jax_orig/build/build.py	2021-07-30 10:18:43.599233082 +0200
++++ jax/build/build.py	2021-08-02 15:58:09.654165000 +0200
+@@ -41,7 +41,7 @@
+ else:
+   from distutils.spawn import find_executable as which
+ # pylint: enable=g-import-not-at-top
+-
++use_default_shell_env = True
```

Comment:
Not required here, this is for Bazel files only, isn't it?

Reply:
true